src/maintainer/adding_pkgs.rst

```diff
@@ -638,17 +638,59 @@ go to the root repository directory and run the
 ``.scripts/run_docker_build.sh`` script.  This requires that you have docker
 installed on your machine.
 
-You need to define an environment variable named ``CONFIG``. Its value must be
-the name of one of the three YAML configuration files in the ``.ci_support``
-directory (either ``linux64``, ``osx64``, or ``win64``). As an example, you can
-invoke the command as follows.
+You need to define an environment variable named ``CONFIG``. Its value must correspond
+the name of one of the YAML configuration file in the ``.ci_support``
+directory (for all practical purposes, only ``linux64`` is used here). As an example, you can
```

Comment:
That's not true.

Reply:
Can you elaborate? I am not sure about windows, but I am 100% this is true for osx. `CONFIG=osx64 ./.scripts/run_docker_build.sh` definitely doesn't work --- it defaults back to `linux64`

Reply:
ahhhh yes you are right, but see isuru's comments below.